app/utils.py

```diff
@@ -954,6 +955,15 @@ def should_show_inline_photo(self, photo_url):
         else:
             return True
 
+    def get_thumbnail_url(self, photo_url):
+        if not photo_url:
+            return None
+        parsed_url = list(urlparse.urlparse(photo_url))
```

Comment:
Users can either specify an external photo URL or upload their local photo to PF when creating a record. So `photo_url` can be an arbitrary external URL. Can you add a check for it?

Reply:
Thanks, added a check in app/results.py (so this doesn't get called for non-uploaded photos).